### PR TITLE
Merge commits from master to inbox

### PR DESCRIPTION
**Pull request by roninopf**

### CODE REVIEW

File: swift-sdk/Internal/IterableInAppHTMLViewController.swift [hunk 1]
```diff
@@ -0,0 +1,223 @@
+//
```
Comment (vbabenkoru) [inline swift-sdk/Internal/IterableInAppHTMLViewController.swift]: Hmm looks like this file was removed in `inbox`.. If it has been refactored into something else, these changes should go to the new files/classes instead.

Comment (roninopf) [reply swift-sdk/Internal/IterableInAppHTMLViewController.swift]: Yeah, this was a mistaken inclusion by me, it just became `IterableHtmlMessageViewController.swift`

**Comment by vbabenkoru** [comment]
Looks like one test failed:
```
    ✗ testMoveToForegroundSyncInterval, Fulfilled inverted expectation "do not sync second time".
    ✗ testInboxOrdering, Test crashed
```
Is this a flaky test or can this be somehow related to the merge?

**Comment by roninopf** [comment]
> 
> 
> Looks like one test failed:
> 
> ```
>     ✗ testMoveToForegroundSyncInterval, Fulfilled inverted expectation "do not sync second time".
>     ✗ testInboxOrdering, Test crashed
> ```
> 
> Is this a flaky test or can this be somehow related to the merge?

Hmmm it sounds like it might be merge related actually, will take a look

**Comment by roninopf** [comment]
Actually I think we're just running into some kind of threading issue; I'll add this build log to our existing ticket but since the build for the PR and my local worked fine, I'll going to say it's fine.

**Pull request merged**